res/css/views/rooms/_ReadReceiptGroup.pcss

```diff
@@ -14,9 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mx_ReadReceiptGroup {
+/* Required against alphabetic order of CSS import */
+:root {
```

Comment:
Otherwise this variable could not be applied to `res/css/views/right_panel/_BaseCard.pcss` (and any CSS files imported before this file).